File: java/HelloWorld.java

public class HelloWorld
{

 public static void main( String[] args)
 {
  TestEnv te= new TestEnv();

  te.checkComplainAndAdjustExpected( 0);

  System.out.println( "Hello World");

  te.checkComplainAndAdjustExpected( 0);

  System.out.println( ""+ te.result());
 }
}

File: c/hello_world.c
#include <stdio.h>

#include "testenv.h"

int main( int argc, char** argv)
{
 
 checkComplainAndAdjustExpected( 0);

 printf( "Hello World\n");

 checkComplainAndAdjustExpected( 0);

 printf( "%s\n", result());

 return 0;
}
